libs/components/autonumeric/src/lib/modules/autonumeric/fixtures/autonumeric.component.fixture.ts

```diff
@@ -19,39 +13,35 @@ import { SkyAutonumericDirective } from '../autonumeric.directive';
 @Component({
   selector: 'sky-autonumeric-directive-test',
   templateUrl: './autonumeric.component.fixture.html',
-  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AutonumericFixtureComponent implements OnInit {
+export class AutonumericFixtureComponent {
   @ViewChild(SkyAutonumericDirective)
   public autonumericDirective: SkyAutonumericDirective | undefined;
 
-  @ViewChild('donationAmountTemplateDriven', { read: NgModel })
-  public donationAmountTemplateDriven: NgModel | undefined;
+  @ViewChild('templateNgModel', { read: NgModel })
+  public templateNgModel!: NgModel;
 
   public autonumericOptions: SkyAutonumericOptions | undefined;
 
-  public formGroup: UntypedFormGroup | undefined;
+  public formGroup: UntypedFormGroup;
 
-  public templateDrivenModel: any = {
-    donationAmount: 1000,
-  };
+  public formControl: AbstractControl;
 
-  public get formControl(): AbstractControl | undefined {
-    return this.formGroup?.get('donationAmount') || undefined;
-  }
+  public templateDrivenDonationAmount: string | number = 1000;
 
   public required = false;
 
   public setUnformatted = false;
 
-  constructor(
-    public changeDetector: ChangeDetectorRef,
-    private formBuilder: UntypedFormBuilder
-  ) {}
+  #formBuilder: UntypedFormBuilder;
 
-  public ngOnInit(): void {
-    this.formGroup = this.formBuilder.group({
+  constructor(formBuilder: UntypedFormBuilder) {
+    this.#formBuilder = formBuilder;
+
+    this.formGroup = this.#formBuilder.group({
       donationAmount: new UntypedFormControl(),
     });
+
+    this.formControl = this.formGroup.controls['donationAmount'];
```

Comment:
The standard way of doing this would be `this.formGroup.get('donationAmount')`.

Reply:
Or you could create the form control right after 39 and assign it to `this.formControl` and then on line 42 just pass `this.formControl` to where you are setting `donationAmount` in the group function.

Reply:
done in [refactor formControl creation](https://github.com/blackbaud/skyux/pull/606/commits/92308bf5459603002b8979c52aa371c0fb10f3a3)

Reply:
I realized this was breaking things. doing the first way causes a warning about donationAmount existing, and the second way does not allow donationAmount to be found. reverted in: [revert form control intialization form](https://github.com/blackbaud/skyux/pull/606/commits/7c5054fa10e9c64cd1386b7e6a725845e0659556)

Reply:
Your change that you reverted wasn't quite what I meant. You can do

```
this.#formBuilder = formBuilder;

    this.formControl = new FormControl('donationAmount');

    this.formGroup = this.#formBuilder.group({ donationAmount: this.formControl });
```

Reply:
Doing it this way is causing the formControl to be marked as invalid on initialization.

Reply:
fixed in: [fix formControl init](https://github.com/blackbaud/skyux/pull/606/commits/43edc78ecbc9dc06f3223e48641439609e24d944)